qtt/algorithms/fitting.py

```diff
@@ -53,10 +53,24 @@ def initFermiLinear(x_data, y_data, fig=None):
         # subtract linear part
         yr = ydata - y
 
-        cc = np.mean(xdata)
+        sigma = xdata.size/250
```

Comment:
Create a subfuction `_estimate_fermi_model_center(...)` (or something like that) out of lines 56 to (roughly) 70